homeassistant/components/media_player/clementine.py

```diff
@@ -0,0 +1,225 @@
+"""
+Support for Clementine Music Player as media player.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.clementine/
+"""
+
+import time
+from datetime import timedelta
+import logging
+import asyncio
```

Comment:
```
    @asyncio.coroutine
    def async_get_media_image(self):
        """Fetch media image of current playing image."""
```

This is a new method introduced by #5754 . The parent method has that signature, as it uses  `yield from` asyncio idiom. Following your comment, I tried without `@asyncio.coroutine`, but it is needed.